rpm/0003-Start-messageserver-on-system-startup-in-case-there-.patch

```diff
@@ -161,13 +161,13 @@ index 00000000..0d583d15
 +[Install]
 +WantedBy=user-session.target
 diff --git a/src/tools/tools.pro b/src/tools/tools.pro
-index d05c3377..689ad320 100644
+index d05c3377..fcecd342 100644
 --- a/src/tools/tools.pro
 +++ b/src/tools/tools.pro
 @@ -1,2 +1,6 @@
  TEMPLATE = subdirs
  SUBDIRS = messageserver
 +
-+contains(DEFINES,USE_ACCOUNTS_QT) {
++packagesExist(accounts-qt5) {
```

Comment:
Hm, also upstream compiles this if found, but that's maybe slightly dangerous as if affects the resulting libraries that much. Though maybe accounts-qt is not found much by accident.

Anyhow, the accounts integration patch does still have one #ifdef USE_ACCOUNTS_QT in servicehandler.cpp and the .spec is building with it. If it's getting automatically used, it should then happen also there?

Reply:
No biggie but guess the explicit opt-in could be nicer, especially since the build process doesn't that well tell what options it's using. Cmake qt6 notes it didn't find accounts-qt6 but that's about it.

Either way good if the acconts-qt usage is consistent. Either fully automatic or then always opt-in.

Reply:
I've reverted this, so it's still using the USE_ACCOUNTS_QT flag.